source/source.go

```diff
@@ -0,0 +1,125 @@
+package source
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/aws/aws-sdk-go-v2/service/sqs"
+	"github.com/aws/aws-sdk-go-v2/service/sqs/types"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+)
+
+type Source struct {
+	sdk.UnimplementedSource
+	config   Config
+	svc      *sqs.Client
+	queueURL string
+}
+
+func NewSource() sdk.Source {
+	return sdk.SourceWithMiddleware(&Source{}, sdk.DefaultSourceMiddleware()...)
+}
+
+func (s *Source) Parameters() map[string]sdk.Parameter {
+	return Config{}.Parameters()
+}
+
+func (s *Source) Configure(ctx context.Context, cfg map[string]string) error {
+	sdk.Logger(ctx).Debug().Msg("Configuring Source Connector.")
+
+	err := sdk.Util.ParseConfig(cfg, &s.config)
+	if err != nil {
+		return fmt.Errorf("failed to parse source config : %w", err)
+	}
+
+	return nil
+}
+
+func (s *Source) Open(ctx context.Context, _ sdk.Position) error {
```

Comment:
I see we're not using the position. Does that mean that the connector will re-read all the messages in the queue?